pages/Contributing and Debugging/Issue-Guidelines.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Issue Guidelines
+---
+
+Inspired by Ghostty's approach to Github issues, we do not allow regular users to open issues anymore.
+
+This page is intended for people who want to open an **issue** on the **Hyprland** repo. Other hyprwm repositories
+still use issues normally.
+
+## Why?
+
+We are volunteers, doing this in our free time. Out of respect, please read this document
+_fully_ before posting an issue _or_ discussion. If you can spend a few minutes reading this,
+it saves us, the developers, much more time overall, so we can deliver fixes and features faster.
+
+Thank you!
+
+## Issue Guidelines
+
+Whatever your issue may be, _do not open an issue_. Instead, open a _discussion_ on the Hyprland repository.
```

Comment:
Again, I think the aggressive tone is not helping anyone. I kinda understand that we don't want the issues tab to be what Turks call a "Yolgeçen hanı" (somewhere anyone of any origin comes and goes) but we also shouldn't treat the users as garbage. Using gentler wording would go much further imo.